1.1/translations/reqs_fr.ttl

```diff
@@ -0,0 +1,762 @@
+                              spec:ConformanceTest ;
+                     spec:testPurpose "Vérifiez la conformité avec cette exigence."@fr ;
+                     skos:definition "Examiner la documentation de la mise en oeuvre pour vérifier que les profils GML sont documentés."@fr ;
+                     skos:prefLabel "GeoSPARQL 1.0 Vérification de la conformité: Profile GML"@fr .
+
+conf10gx:query-functions rdf:type 
+                                  spec:ConformanceTest ;
+                         spec:testPurpose "Vérifiez la conformité avec cette exigence."@fr ;
+                         skos:definition "Vérifiez qu'un ensemble de requêtes SPARQL impliquant chacune des fonctions suivantes renvoie le résultat correct pour un ensemble de données de test lors de l'utilisation de la sérialisation et de la version spécifiée:  geof:distance, geof:buffer, geof:convexHull, geof:intersection, geof:union, geof:difference,geof:symDifference, geof:envelope and geof:boundary."@fr ;
+                         skos:prefLabel "GeoSPARQL 1.0 Vérification de la conformité: Fonctions Requêtes"@fr .
+
+conf10gx:srid-function rdf:type 
+                                spec:ConformanceTest ;
+                       spec:testPurpose "Vérifiez la conformité avec cette exigence."@fr ;
+                       skos:definition "Vérifier qu'une requête SPARQL impliquant la fonction geof:getSRID renvoie le résultat correct sur des données de test lors de l'utilisation de la sérialisation et de la version spécifiée."@fr ;
+                       skos:prefLabel "GeoSPARQL 1.0 Vérification de la conformité: Fonctions SRID"@fr .
+
+conf10gx:wkt-axis-order rdf:type 
+                                 spec:ConformanceTest ;
+                        spec:testPurpose "Vérifiez la conformité avec cette exigence."@fr ;
+                        skos:definition "Vérifier que les requêtes impliquant les valeurs geo:wktLiteral renvoient le résultat correct sur des données de test."@fr ;
+                        skos:prefLabel "GeoSPARQL 1.0 Vérification de la conformité: Ordre des Axes WKT"@fr .
+
+conf10gx:wkt-literal rdf:type 
+                              spec:ConformanceTest ;
+                     spec:testPurpose "Vérifiez la conformité avec cette exigence."@fr ;
+                     skos:definition "Vérifier que les requêtes impliquant les valeurs geo:wktLiteral renvoient le résultat correct sur des données de test."@fr ;
+                     skos:prefLabel "GeoSPARQL 1.0 Vérification de la conformité: Littéral WKT"@fr .
+
+conf10gx:wkt-literal-default-srs rdf:type 
+                                          spec:ConformanceTest ;
+                                 spec:testPurpose "Vérifiez la conformité avec cette exigence."@fr ;
+                                 skos:definition "Vérifier que les requêtes impliquant les valeurs geo:wktLiteral sans URI de système de référence spatiale encodé explicite renvoient le résultat correctsur des données de test."@fr ;
```

Comment:
- turn "explicite" to "explicitement"
- fix typo "correct sur"

Reply:
Rewrote.